data/typelibrary/io-1.0.0/typelib/revpi/IORevPiAIO.fbt

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <FBType Name="IORevPiAIO" Comment="Analog I/O module for Revolution Pi by KUNBUS GmbH">
-	<Identification Standard="61499-2" Description="Copyright (c) 2024 Monika Wenger
&#10; 
&#10;This program and the accompanying materials are made
&#10;available under the terms of the Eclipse Public License 2.0
&#10;which is available at https://www.eclipse.org/legal/epl-2.0/
&#10;
&#10;SPDX-License-Identifier: EPL-2.0" >
+	<Identification Standard="61499-2" Description="Copyright (c) 2024 Monika Wenger&#10;&#10;This program and the accompanying materials are made&#10;available under the terms of the Eclipse Public License 2.0&#10;which is available at https://www.eclipse.org/legal/epl-2.0/&#10;&#10;SPDX-License-Identifier: EPL-2.0" >
```

Comment:
Just asking the stupid question: did you delete the the linefeed characters manually?

The linefeed characters already appear in https://github.com/eclipse-4diac/4diac-ide/raw/freeze/data/typelibrary/signalprocessing-1.0.0/typelib/DualHysteresis.fbt

but not in https://github.com/eclipse-4diac/4diac-ide/blob/freeze/data/typelibrary/io-1.0.0/typelib/IB.fbt where they were escaped to &#13;

Reply:
I opened these files within notepad++ within the statusbar I can say use Linux linefeed this changes the whole file, I only need to save it then. But I also removed all new lines within this description, since I was not sure if one of these steps is enough. At least I did not see the ^M within any editor.